metrics/windows/interface.go

```diff
@@ -62,7 +62,7 @@ func NewInterfaceGenerator(interval time.Duration) (*InterfaceGenerator, error)
 	names := []string{}
 	for ai = first; ai != nil; ai = ai.Next {
 		name, err := windows.AnsiBytePtrToString(&ai.Description[0])
-		if err == nil {
+		if err == nil && name != "" {
```

Comment:
This code is used for collecting metrics, not used when registering host information, right?

Reply:
I added same code to `Generate` in spec. Thank you.